From a8e75bb73c6f8ce94c4f598a2f4763b12188991c Mon Sep 17 00:00:00 2001
From: Lepton Wu <lepton@chromium.org>
Date: Wed, 2 Jun 2021 18:00:06 -0700
Subject: [PATCH] Revert "virgl: Cache depth and stencil buffers"

This reverts commit d245d7b6b8bb75622dd43fb845f60d55ad3ebc89.

It broke various dEQP EGL tests because the reused resource
at host side could have a different sample_count with what we want.
The example of tests get broken:

dEQP-EGL.functional.color_clears.single_context.gles2.rgba8888_window

Signed-off-by: Lepton Wu <lepton@chromium.org>
Reviewed-by: Rohan Garg <rohan.garg@collabora.com>
Reviewed-by: Chia-I Wu <olvaffe@gmail.com>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/11152>
---
 src/gallium/winsys/virgl/drm/virgl_drm_winsys.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/gallium/winsys/virgl/drm/virgl_drm_winsys.c b/src/gallium/winsys/virgl/drm/virgl_drm_winsys.c
index 1dd7b001925..dfa145a73f7 100644
--- a/src/gallium/winsys/virgl/drm/virgl_drm_winsys.c
+++ b/src/gallium/winsys/virgl/drm/virgl_drm_winsys.c
@@ -63,7 +63,6 @@ static inline boolean can_cache_resource(uint32_t bind)
           bind == VIRGL_BIND_VERTEX_BUFFER ||
           bind == VIRGL_BIND_CUSTOM ||
           bind == VIRGL_BIND_STAGING ||
-          bind == VIRGL_BIND_DEPTH_STENCIL ||
           bind == 0;
 }
 
-- 
2.32.0.rc1.229.g3e70b5a671-goog

